ONSdigital/eq-author-app
# Remove scroll bar from comment textArea's

### What is the context of this PR?

Made the comment textArea expand by removing the row props

Needed to add a scroll bar to the pane itself though otherwise it was impossible to get to the bottom of the container to hit the add button.

**Help**
Looks kinda naff when you add another comment, need to show Joe

### How to review 

1. Create questionnaire
2. Add a comment
3. Check if there's a scroll bar
